sdk/python/feast/infra/online_stores/online_store.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2021 The Feast Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from abc import ABC, abstractmethod
+from datetime import datetime
+from typing import Any, Callable, Dict, List, Optional, Tuple, Union
+
+from feast import FeatureTable
+from feast.feature_view import FeatureView
+from feast.protos.feast.types.EntityKey_pb2 import EntityKey as EntityKeyProto
+from feast.protos.feast.types.Value_pb2 import Value as ValueProto
+from feast.repo_config import RepoConfig
+
+
+class OnlineStore(ABC):
+    """
+    OnlineStore is an object used for all interaction between Feast and the service used for online storage of
+    features.
+    """
+
+    @staticmethod
+    @abstractmethod
+    def online_write_batch(
```

Comment:
Can we please add comments that describe these two abstract methods?

Reply:
Added